modules/gui.py

```diff
@@ -664,6 +667,18 @@ def refresh_fonts(self):
         self.impl.refresh_font_texture()
         self.type_label_width = None
 
+    def save_filters(self):
+        if self.filters:
```

Comment:
Are you sure about this? I feel like this would mean that if you close the program once with filters they get saved, then close and reopen, they get restored, but then if you remove them and close without filters it will not update the filters file... so you wont be able to remove filters permanently...

Reply:
True, my bad. Premature optimization in all it's glory.